src/pages/index.js

```diff
@@ -134,6 +135,12 @@ export default props => {
           href='/masonry'
           zoom={1/4}
         />
+        <Card
+          title='Holy Grail'
+          component={HolyGrail}
+          href='/holy-grail'
+          zoom={1/4}
```

Comment:
Pretty sure the `zoom` here is making this one look smaller than the header/footer one – used it on the Masonry example to help it fit better in the card, but don't think it's needed for this example
```suggestion
```